msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md

```diff
@@ -187,9 +187,9 @@ Your bot can [send proactive messages](/azure/bot-service/bot-builder-howto-proa
 
 ## Code sample
 
-| **Sample Name** | **Description** | **.NET** | **Node.js** |
-|---------------|--------------|--------|-------------|--------|
-| Proactive installation of app and sending proactive notifications | This sample shows how you can use proactive installation of app for users and send proactive notifications by calling Microsoft Graph APIs. | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/graph-proactive-installation/csharp) | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/graph-proactive-installation/nodejs) |
```

Comment:
This is not the change which I requested. We need to add new entry in TOC for Sample code under **Utilize Teams data with Microsoft Graph** section.
![image](https://user-images.githubusercontent.com/31851992/126746945-60abdf07-742e-43e1-b174-3a0808ac0859.png)

Reply:
@Wajeed-msft, added TOC entry for code samples.